scripts/test/TOFTOF/TOFTOFGUITest.py

```diff
@@ -0,0 +1,133 @@
+from __future__ import (absolute_import, division, print_function)
+
+from reduction_gui.reduction.toftof.toftof_reduction import TOFTOFScriptElement, OptionalFloat
+from reduction_gui.widgets.toftof.toftof_setup import TOFTOFSetupWidget
+import mantid
+from PyQt4 import QtGui
+
+import unittest
+try:
+    from unittest import mock
+except ImportError:
+    import mock
+
+
+class TOFTOFScriptElementTest(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls):
+        cls.app = QtGui.QApplication([])
+
+    def setUp(self):
+        self.settingsMock = mock.Mock()
+        facility_name = "name of facility"
+        self.settingsMock.facility_name = facility_name
```

Comment:
Why not `self.settingsMock.facility_name = "name of facility"`? Ditto for instrument name below.